parquet-column/src/test/java/org/apache/parquet/schema/TestMessageType.java

```diff
@@ -173,4 +173,16 @@ public void testIDs() throws Exception {
     assertEquals(schema, schema2);
     assertEquals(schema.toString(), schema2.toString());
   }
+
+  @Test
+  public void testQuotedIdentifiers() throws Exception {
+    MessageType schema = new MessageType("test",
+        new PrimitiveType(REQUIRED, BINARY, "foo bar"),
+        new PrimitiveType(REQUIRED, BINARY, "foo()`bar"),
```

Comment:
Do we need to support ` in names? Is it supported elsewhere, like in Hive?

It doesn't seem worth the added complexity in the tokenizer to me.